.github/CONTRIBUTING.md

```diff
@@ -208,11 +208,13 @@ There will be times that we really like your proposed changes and we’ll finish
 
 The Core Contributors team consists of one member of Umbraco HQ, [Sebastiaan][Sebastiaan], who gets assistance from the following community members who have committed to volunteering their free time:
 
-- [Nathan Woulfe][Nathan Woulfe]
-- [Joe Glombek][Joe Glombek]
-- [Laura Weatherhead][Laura Weatherhead]
+- [Busra Sengul][Busra Sengul]
+- [Emma Garland][Emma Garland]
+- [George Bidder][George Bidder]
+- [Jason Elkin][Jason Elkin]
+- [Laura Neto][Laura Neto]
```

Comment:
Missing Kyle in here I think😉 ?

![image](https://user-images.githubusercontent.com/1769475/226494954-0ee35183-2e2e-4104-bcc7-d72f5cf5e32e.png)